app/controllers/api/v1/reserved_flights_controller.rb

```diff
@@ -0,0 +1,74 @@
+class Api::V1::ReservedFlightsController < ApplicationController
+  before_action :set_reserved_flight, only: %i[show edit update destroy]
+
+  # GET /reserved_flights or /reserved_flights.json
+  def index
+    @reserved_flights = ReservedFlight.all
+    render json: @reserved_flights
+  end
+
+  # GET /reserved_flights/1 or /reserved_flights/1.json
+  def show; end
+
+  # GET /reserved_flights/new
+  def new
+    @reserved_flight = ReservedFlight.new
+  end
+
+  # GET /reserved_flights/1/edit
+  def edit; end
+
+  # POST /reserved_flights or /reserved_flights.json
+  def create
+    @reserved_flight = ReservedFlight.new(reserved_flight_params)
+
+    respond_to do |format|
+      if @reserved_flight.save
+        format.html do
+          redirect_to reserved_flight_url(@reserved_flight), notice: 'Reserved flight was successfully created.'
+        end
+        format.json { render :show, status: :created, location: @reserved_flight }
+      else
+        format.html { render :new, status: :unprocessable_entity }
+        format.json { render json: @reserved_flight.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # PATCH/PUT /reserved_flights/1 or /reserved_flights/1.json
+  def update
+    respond_to do |format|
+      if @reserved_flight.update(reserved_flight_params)
+        format.html do
+          redirect_to reserved_flight_url(@reserved_flight), notice: 'Reserved flight was successfully updated.'
+        end
+        format.json { render :show, status: :ok, location: @reserved_flight }
+      else
+        format.html { render :edit, status: :unprocessable_entity }
+        format.json { render json: @reserved_flight.errors, status: :unprocessable_entity }
+      end
+    end
+  end
+
+  # DELETE /reserved_flights/1 or /reserved_flights/1.json
+  def destroy
+    @reserved_flight.destroy
```

Comment:
Missing error handling here, what happens if it does not work the way you expect it to?